migrations/sql/V2024.03.11.10.00__add_project_contact_first_last_name_column_and_data.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE project_contact
+    ADD COLUMN IF NOT EXISTS first_name varchar,
+    ADD COLUMN IF NOT EXISTS last_name varchar;
```

Comment:
first_name and last_name may require a character length.

Reply:
Sounds good, they have been adjusted to have character length 200 now.